### ISSUE TITLE
Is it possible to get the learning rate from the trainer?

### ISSUE THREAD

**Issue by vedantroy** [lucidrains/imagen-pytorch#134]
Hi! I'd like to log the learning rate to wandb (I only have a single learning rate, since I'm training a single unet), but it seems to be hidden somewhere in the trainer. Is there maybe an easy way to do this?

**Comment by lucidrains** [comment]
@vedantroy sure! let me know if https://github.com/lucidrains/imagen-pytorch/commit/91976013aa6ae0f921ed3625c5cc7d0399e2f90c#diff-db403ff86c6fb0812b3ce92a98fb7d76cc888c28dfa3e0a5421bde05febd47bbR357 works

**Comment by vedantroy** [comment]
Thanks, will try it out!
